torchao/quantization/__init__.py

```diff
@@ -15,6 +15,12 @@
     OTHER_AUTOQUANT_CLASS_LIST,
     autoquant,
 )
+from torchao.prototype.quantization.autoquant_v2 import (
```

Comment:
it's unexpected to have something here which is a prototype, IMO either not have it in prototype or not have it here